app/controllers/admin/loaned_books_controller.rb

```diff
@@ -0,0 +1,66 @@
+class Admin::LoanedBooksController < Admin::BaseController
+  before_action :assign_new_loan, only: :create
+  before_action :find_book, only: :create
+  before_action :check_quantity_add, only: :create
```

Comment:
dồn thành 1 line dc cu nè
```suggestion
  before_action :assign_new_loan, :find_book, :check_quantity_add, only: :create
```

Reply:
cơ mà muốn mượn sách phải login và ở đây chị thấy namesapce là admin, thì chị lại ko thấy em check role là admin với đã login nhỉ

Reply:
Dạ chị, em check pull của Công rồi em thêm vào.

Reply:
Cái này em check bên chỗ base_controller mà nó kế thừa luôn chị nha.